huaweicloud/services/evs/resource_huaweicloud_evs_volume.go

```diff
@@ -655,3 +721,30 @@ func CloudVolumeRefreshFunc(c *golangsdk.ServiceClient, volumeId string) resourc
 		return response, "ERROR", nil
 	}
 }
+
+func RefreshVolumeStatusFunc(c *golangsdk.ServiceClient, volumeId string) resource.StateRefreshFunc {
```

Comment:
it's unnecessary to export this function.
```suggestion
func refreshVolumeStatusFunc(c *golangsdk.ServiceClient, volumeId string) resource.StateRefreshFunc {
```